TeamCharles/workforce-management
# 34-training-program-index

## Status
**READY**

## Documentation

[X] I have fully documented features in this PR's code. Please put an `X` in the box to confirm.

[ ] This PR does **NOT** require documentation.

## Description
Provide a view for the user that will display a list of all Training Program names and descriptions.

## Resolves Issue Number
Resolves #34 

## Deploy Notes

```sh
git pull --prune
git checkout 34-training-program-index
dotnet run
```

## Steps to Test or Reproduce

1.  Navigate to localhost/TrainingProgram/Index
1. Confirm that a list of Training Programs appears

## Impacted Files in Application

* Views/TrainingProgram/Index
* ViewModels/TrainingProgram/TrainingProgramIndex
* Controllers/TrainingProgramController
